Dockerfile

```diff
@@ -27,13 +27,12 @@ ARG BUILD_VERSION=v3.0.0-1-g6666666
 ARG GIT_COMMIT=0
 RUN echo -e "module.exports = { version: '$BUILD_VERSION', revision: '$GIT_COMMIT' }" > ./version.js
 
-
 FROM base AS development 
 
 # Temporarily disable the postinstall NPM script
-RUN npm pkg set scripts.postinstall="echo no-postinstall"
-RUN npm i --include dev
-RUN npm run build
+RUN npm pkg set scripts.postinstall="echo no-postinstall" \
```

Comment:
Does the BASH style newline demarcation work in Docker? I presume it is doing some pre-processing on the content before sending it for execution, I'm just not 100%.